src/topology/continuous_function/units.lean

```diff
@@ -0,0 +1,94 @@
+/-
+Copyright (c) 2022 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import topology.continuous_function.compact
+import analysis.normed_space.units
+import algebra.algebra.spectrum
+
+/-!
+# Units of continuous functions
+
+This file concerns itself with `C(α, β)ˣ` and `C(α, βˣ)` when `α` is a topological space
+and `β` is a normed ring.
+-/
+
+namespace continuous_map
+
+section units
+
+section normed_ring
+
+variables {α : Type*} [topological_space α] {β : Type*} [normed_ring β]
+
+/-- Equivalence between continuous maps into the units of a normed ring the
+the units of the ring of continuous functions. -/
+@[simps]
+def units_lift : C(α, βˣ) ≃ C(α, β)ˣ :=
```

Comment:
This only needs `[monoid β] [has_continuous_mul β]` not `normed_ring`

Reply:
ah thanks, I forgot to go looking for the appropriate generalization.